01/app.js

```diff
@@ -1 +1,11 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const liEl = document.qetElementsByClassName ('comments__item comments__item--newest');
+
+
+if (liEl) {
+    const dataEl = liEl.querySelectorAll('data-info');
```

Comment:
Ponieważ możemy przeszukiwać tylko 1 element to musimy go wskazać przez indeks czyli `liEl[0].querySelectorAll()`
Dodatkowo wyszukiwanie po atrybucie to nawiasy kwadratowe tj. `.querySelectorAll("[data-info]")`